hadoop-ozone/ozone-manager/src/main/resources/webapps/ozoneManager/om-overview.html

```diff
@@ -90,4 +90,78 @@ <h2>Meta-Data Volume Information</h2>
         <td>{{$ctrl.overview.jmx.RocksDbDirectory}}</td>
     </tr>
     </tbody>
-</table>
\ No newline at end of file
+</table>
+
+<!-- Only display Deletion Progress on OM leader -->
+<div ng-show="$ctrl.role.Role.trim() === 'LEADER'">
+  <h2>Deletion Progress [since {{$ctrl.overview.jmx.MetricsResetTimeStamp * 1000 | date:'yyyy-MM-dd HH:mm:ss'}}]</h2>
+  <table class="table table-bordered table-striped" class="col-md-6">
+      <thead>
+      <tr>
+          <th>Total Reclaimed Key Size(Replicated)</th>
+          <th>No. of Keys Reclaimed</th>
+      </tr>
+      </thead>
+      <tbody>
+      <tr>
+          <td>{{$ctrl.formatBytes($ctrl.overview.jmx.ReclaimedSizeInInterval)}}</td>
+          <td>{{$ctrl.overview.jmx.KeysReclaimedInInterval}}</td>
+      </tr>
+      </tbody>
+  </table>
+  <div class="mt-4">
+      <h3>
+          Deletion Service
+          <span class="badge ml-2" ng-class="{'badge-success': $ctrl.overview.jmx.KdsServiceState == 1, 'badge-secondary': $ctrl.overview.jmx.KdsServiceState == 0}">
+              {{$ctrl.overview.jmx.KdsServiceState == 1 ? 'Running' : 'Idle'}}
+          </span>
+      </h3>
+      <div class="mt-3">
+          <div class="row mb-2" ng-if="$ctrl.overview.jmx.KdsLastRunTimestamp">
+              <div class="col-md-3">Last Run:</div>
+              <div class="col-md-9">{{$ctrl.convertMsToTime($ctrl.Date.now() - $ctrl.overview.jmx.KdsLastRunTimestamp)}} ago</div>
+          </div>
+          <div class="row" ng-if="$ctrl.overview.jmx.KdsNextRunTimestamp">
+              <div class="col-md-3">Next Run:</div>
+              <div class="col-md-9">in {{$ctrl.convertMsToTime($ctrl.overview.jmx.KdsNextRunTimestamp - $ctrl.Date.now())}}</div>
+          </div>
+      </div>
+      <div class="mt-3">
+          <div class="col-md-12">
+              <button class="btn btn-sm btn-link p-0" ng-click="lastRunDetailsVisible = !lastRunDetailsVisible">
+                  {{ lastRunDetailsVisible ? 'Last Run Details(-)' : 'Last Run Details(+)' }}
+              </button>
+          </div>
+          <div ng-show="lastRunDetailsVisible">
+              <table class="table table-sm table-bordered mt-2">
+                  <thead>
+                      <tr>
+                          <th>Store</th>
+                          <th>Reclaimed Size</th>
+                          <th>#Reclaimed Keys</th>
+                          <th>#Iterated Keys</th>
+                          <th>#NotReclaimable Keys</th>
```

Comment:
NotReclaimable keys makes sense in context of snapshots. We could change it to
```suggestion
                          <th>#NotReclaimable Keys (Referred by snapshots)</th>
```